CHANGELOG.md

```diff
@@ -1,5 +1,59 @@
 # Optimizely Ruby SDK Changelog
 
+## 5.0.0-beta
+April 28th, 2023
+
+### New Features
+
+The 5.0.0-beta release introduces a new primary feature, [Advanced Audience Targeting]( https://docs.developers.optimizely.com/feature-experimentation/docs/optimizely-data-platform-advanced-audience-targeting) enabled through integration with [Optimizely Data Platform (ODP)](https://docs.developers.optimizely.com/optimizely-data-platform/docs)
+([#303](https://github.com/optimizely/ruby-sdk/pull/303),
+[#306](https://github.com/optimizely/ruby-sdk/pull/306),
+[#308](https://github.com/optimizely/ruby-sdk/pull/308),
+[#310](https://github.com/optimizely/ruby-sdk/pull/310),
+[#311](https://github.com/optimizely/ruby-sdk/pull/311),
+[#312](https://github.com/optimizely/ruby-sdk/pull/312),
+[#314](https://github.com/optimizely/ruby-sdk/pull/314),
+[#316](https://github.com/optimizely/ruby-sdk/pull/316),
+[#323](https://github.com/optimizely/ruby-sdk/pull/323),
+[#324](https://github.com/optimizely/ruby-sdk/pull/324)).
```

Comment:
@jaeopt was telling me to limit the number of PRs to 4-5 main ones. I wonder if that applies to here as well?